components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag

```diff
@@ -43,9 +43,9 @@
 			else if( option === "avgprocessduration"){
 				response.content = processBarChartAvgTime;
 			}
-			else{
```

Comment:
This code snippet has been moved to reportController.jag and the formatting issue is fixed there.
Fixed with commit 130406a